### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by SimoneMartino98**
Hi everybody;
In order to solve some minor issues I made this pull request that contain some changes. 

### Solved #8 
Added DEV installation part to the README; @andrewtarzia if you want to check this part because it is the only part that I wrote by myself.

### Solved #14 
Changed the sentence in the ``onion-clustering`` docs part according to the one suggested by @matteobecchi (let me know if it is ok, I've simply copied and pasted it).

### Solved #30 
Added ``onion-clustering`` to the group of Dynsight public dependencies and removed it from the DEV ones. Now Onion should be installed during the Dynsight installation procedure (``pip install dynsight``).

### New mainteiners
Added me and @matteobecchi as new maintainers, with @andrewtarzia, of the Dynsight project (in both README.rst and pyproject.toml).

After that new pull requests will follow with other bigger updates.
thank you

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -10,6 +12,20 @@ To get ``dynsight``, you can install it with pip::
     $ pip install dynsight
 
 
+DEV installation
```
Comment (andrewtarzia) [inline README.rst:15]: Make it "Developer Setup"

File: README.rst [hunk 2]
```diff
@@ -10,6 +12,20 @@ To get ``dynsight``, you can install it with pip::
...
+============
```
Comment (andrewtarzia) [inline README.rst:16]: Make sure the "=" line always matches the length of the title above it.

File: README.rst [hunk 3]
```diff
@@ -10,6 +12,20 @@ To get ``dynsight``, you can install it with pip::
...
+
+If you want to setup a development version of ``dynsight`` you can follow this simple steps:
```
Comment (andrewtarzia) [inline README.rst:18]: Remove this line, and just have the numbered list.

File: README.rst [hunk 4]
```diff
@@ -10,6 +12,20 @@ To get ``dynsight``, you can install it with pip::
...
+    1. Fork the ``dynsight`` repository on your github account by clicking the ``Fork`` button in the github page.
+    2. Clone your forked repository by using ``git clone <YOUR_LINK>`` (fill properly).
+    3. Create a new virual environment or ``conda`` or ``mamba`` (reccomended) using Python 3.10 version.
+    4. Install ``just`` (https://github.com/casey/just).
+    5. Activate the virtual environment created and run ``just dev`` within the cloned repository.
+
```
Comment (andrewtarzia) [inline README.rst:24]: Maybe change the above to something like (my rationale is that you do not need to tell people how to clone a repository):

```
#. Install `just`_.
#. Fork the ``dynsight`` repository
#. In a new virtual environment run using Python 3.10::

    $ just dev

#. Run code checks::

    $ just check
  


.. _`just`: https://github.com/casey/just
```

**Review by andrewtarzia — CHANGES_REQUESTED** [review]

**Comment by SimoneMartino98** [comment]
All requests should be made. Let me know if there are any other changes.

File: README.rst (hunk 4 above)
Comment (andrewtarzia) [reply README.rst:26]: Check formatting.

**Review by matteobecchi — APPROVED** [review]
Changes regarding the onion pytest are OK!

File: docs/source/onion.rst [hunk 5]
```diff
@@ -103,6 +103,6 @@ Example: clustering of multivariate time-series
 
   OnionMulti <_autosummary/dynsight.onion.OnionMulti.rst>
 
-The code for performing clustering on multi time-series can be found in ``examples/onion_multi.py``. The only difference in the usage is that when you run the script you have to specify all the input files, each one containing a different variable:
+The code for performing clustering on multi time-series can be found in ``examples/onion_multi.py``. The only differences that the input file has to be an .npy or .npz containing a numpy.ndarray with shape (D, N, T), where D is the number of variables (2 or 3). It is then called with
```
Comment (matteobecchi) [inline docs/source/onion.rst:106]: Should it be "The only difference is that" instead of "The only differences that"?

**Pull request merged**